interface1.js

```diff
@@ -421,14 +421,23 @@ window.addEventListener('load', function(){
 
 		//Get input to GEN.
 		var pString = spotForm.inputToGen.value;
-
 		// Get the code that is in the stree textarea
-		var treeCode = spotForm.sTree.value
+		var treeCode = spotForm.sTree.value;
 		// if code has been generated, then ignore pString in GEN
 		if(treeCode !== "{}") {
 			pString = "";
 		}
-
+		var doubleInputWarningMsg = "Inputs were provided on both the Manual tab and the Automatic tab of Gen: Inputs. The candidate set will be created using inputs on the tab that is currently visible. Inputs that are not currently displayed will be ignored.";
+		if(document.getElementById('inputOptions').style.display == 'block') {
+			if (spotForm.inputToGen.value != "" || treeCode != "{}"){
+				displayWarning(doubleInputWarningMsg);
+			}
+			pString = "";
+		}else{
+			if (spotForm.inputToGenAuto.value != ""){
```

Comment:
This is not the best field to check the value of, because it's just the first terminal string field, rather than the actual list of trees. Try checking the value of getAutoStreeList() instead.

The problem to address: If the user makes some trees in the Automatic tab, then deletes the list of terminals out of inputToGenAuto for some reason, then build a tree or trees in the Manual tab, and clicks Get results, then the warning does not display.